account_voucher_killer/__manifest__.py

```diff
@@ -19,16 +19,16 @@
 #
 ##############################################################################
 
-{'name': 'Accounting voucher killer',
- 'version': '8.0.1.0.0',
+{'name': 'Accounting Payment Access',
+ 'version': '10.0.1.0.0',
  'category': 'other',
  'author': "Camptocamp,Odoo Community Association (OCA)",
  'website': 'http://www.camptocamp.com',
```

Comment:
Change `'website' : https://github.com/OCA/bank-payment`
May be you should define a meaningful category.